setup.py

```diff
@@ -1,7 +1,17 @@
 import os
+import sys
 import re
 from setuptools import setup, find_packages
 
+if sys.platform == 'win32':
```

Comment:
I think we can import ```ctypes``` without checking platform type. ctype library available for all platforms and code will be more readable in this case